### PR TITLE
[apptools-windows] Add tests to test enable crosswalk download on windows

### PR DESCRIPTION
**Pull request by yunxliu**
Failure analysis:
https://crosswalk-project.org/jira/browse/XWALK-5563

Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: [Windows]
Unit test result summary: pass 0, fail 2, block 0

https://crosswalk-project.org/jira/browse/XWALK-5288
BUG=https://crosswalk-project.org/jira/browse/XWALK-5556

### CODE REVIEW

File: apptools/apptools-windows-tests/apptools/create_package.py [hunk 1]
```diff
@@ -71,5 +71,23 @@ def test_create_package_missing_icon_startUrl(self):
...
         self.assertEquals(return_code, 0)
 
+    def test_create_package_canary(self):
+        comm.setUp()
+        os.chdir(comm.XwalkPath)
+        comm.clear("org.xwalk.test")
+        os.mkdir("org.xwalk.test")
+        os.chdir('org.xwalk.test')
+        cmd = comm.HOST_PREFIX + comm.PackTools + \
+            "crosswalk-pkg --platforms=windows --crosswalk=canary " + comm.ConstPath + "/../testapp/create_package_missing_icon_startUrl/"
```
Comment (Honry) [inline apptools/apptools-windows-tests/apptools/create_package.py]: Why use this testapp, missing_icon_startUrl?

Comment (yunxliu) [inline apptools/apptools-windows-tests/apptools/create_package.py]: It also can use the testapp create_package_basic, and the testapp only have an html file is also can be used.

Comment (Honry) [inline apptools/apptools-windows-tests/apptools/create_package.py]: I just think that the name of this test app may make confuse here.

**Comment by Honry** [comment]
LGTM

**Pull request merged**